### PR TITLE
merge content of stream-introduction.md

### PR DESCRIPTION
**Pull request by naferx**
Ref #23052

### CODE REVIEW

File: akka-docs/src/main/paradox/java/stream/stream-introduction.md [hunk 1]
```diff
@@ -1,75 +1 @@
...
- * The top-down learners may want to peruse the @ref:[Design Principles behind Akka Streams](../general/stream/stream-design.md) at this
-point.
- * The bottom-up learners may feel more at home rummaging through the
-@ref:[Streams Cookbook](stream-cookbook.md).
- * For a complete overview of the built-in processing stages you can look at the
-table in @ref:[stages overview](stages-overview.md)
- * The other sections can be read sequentially or as needed during the previous
-steps, each digging deeper into specific topics.
\ No newline at end of file
+../../scala/stream/stream-introduction.md
```
Comment (raboof) [inline akka-docs/src/main/paradox/java/stream/stream-introduction.md]: I'm sorry, this is a bit confusing in the GitHub UI, but this is still not a symlink, just a file with `../../scala/stream/stream-introduction.md` in it... you can test the documentation output with 'sbt paradox', then you'll see this page doesn't have the expected contents but just this filename.

I'll add a short guide on creating symlinks to #23052 tomorrow!

**Review by raboof — DISMISSED** [review]

File: akka-docs/src/main/paradox/java/stream/stream-introduction.md (hunk 1 above)
Comment (naferx) [inline akka-docs/src/main/paradox/java/stream/stream-introduction.md]: Hehe, I haven't realized. Confusion was mine

Comment (naferx) [inline akka-docs/src/main/paradox/java/stream/stream-introduction.md]: Hehe, I didn't realize that. Confusion was mine

**Review by ktoso — APPROVED** [review]
LGTM, now it's a proper symlink :)

**Comment by ktoso** [comment]
Thanks @naferx !

**Pull request merged**